packages/examples/src/scripts/counter-client.ts

```diff
@@ -13,23 +13,33 @@
  * SPDX-License-Identifier: EPL-2.0 OR W3C-20150513
  ********************************************************************************/
 
-import { Helpers } from "@node-wot/core";
+import { Servient, Helpers } from "@node-wot/core";
+import { HttpClientFactory } from "@node-wot/binding-http";
+import { CoapClientFactory } from "@node-wot/binding-coap";
 import { ThingDescription } from "wot-typescript-definitions";
 
-let WoTHelpers: Helpers;
+// create Servient and add HTTP/CoAP binding
+const servient = new Servient();
+servient.addClientFactory(new HttpClientFactory());
+servient.addClientFactory(new CoapClientFactory());
+
+const wotHelper = new Helpers(servient);
 
 function getFormIndexForDecrementWithCoAP(thing: WoT.ConsumedThing): number {
-    const forms = thing.getThingDescription().actions.decrement.forms;
-    for (let i = 0; i < forms.length; i++) {
-        if (/^coaps?:\/\/.*/.test(forms[i].href)) {
-            return i;
+    const forms = thing.getThingDescription().actions?.decrement.forms;
+    if (forms !== undefined) {
+        for (let i = 0; i < forms.length; i++) {
+            if (/^coaps?:\/\/.*/.test(forms[i].href)) {
+                return i;
+            }
         }
     }
     // return formIndex: 0 if no CoAP target IRI found
     return 0;
 }
 
-WoTHelpers.fetch("coap://localhost:5683/counter")
+wotHelper
+    .fetch("coap://localhost:5683/counter")
```

Comment:
Using `WoTHelpers` directly as we used to do does no longer work..

```
let WoTHelpers: Helpers;
WoTHelpers
    .fetch("coap://localhost:5683/counter") ...
```

It errors with

`error TS2454: Variable 'WoTHelpers' is used before being assigned.`

Are there alternatives? Once we add the imports etc also the generated JS code gets more complicated...

Reply:
This is tricky... two alternatives:
- we go down to the bottom of this issue. Over the years the module and typing system has been improved and the problems that we have might be easy to fix now. 
- disable checking this rule just for the examples.

Reply:
Mhh, I think we might go with disabling the check... 
Please speak up if someone is against it..

Reply:
I changed 

`let WoTHelpers: Helpers;`
to
`let WoTHelpers!: Helpers;`

in the TS file which did the trick. The resulting/generated JS does not change...